File: baekjoon/bronze/level1/test_1357/java/Main.java
package bronze.level1.test_1357.java;

import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        StringBuilder input = new StringBuilder(sc.nextLine()).reverse();
        String[] arr = input.toString().split(" ");

        int sum = Integer.parseInt(arr[0]) + Integer.parseInt(arr[1]);
        StringBuilder sb = new StringBuilder(String.valueOf(sum));
        System.out.print(Integer.parseInt(sb.reverse().toString()));
    }
}

File: baekjoon/bronze/level1/test_1357/kotlin/Main.kt
package baekjoon.bronze.level1.test_1357.kotlin

fun main() {
    val arr = readln().split(" ")
    val a = StringBuilder(arr[0]).reverse().toString().toInt()
    val b = StringBuilder(arr[1]).reverse().toString().toInt()

    val sum = a + b
    print(StringBuilder(sum.toString()).reverse().toString().toInt())
}
